summariser/api/server.py

```diff
@@ -1,26 +1,29 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Query
```

Comment:
Remove what you don't use
```suggestion
from fastapi import FastAPI
```